package.json

```diff
@@ -28,7 +28,9 @@
   },
   "dependencies": {
     "passport": "^0.3.2",
-    "underscore": "^1.8.2"
+    "underscore": "^1.8.2",
+    "eslint": "^2.5.3",
+    "eslint-config-loopback": "^1.0.0"
   },
   "devDependencies": {
     "loopback": "^2.14.0",
```

Comment:
In the scripts section, add `"lint": "eslint ."` and also `"posttest": "npm run lint"`